### DISCUSSION TITLE
trojan?

### DISCUSSION THREAD

**Discussion by debifox** [uBlockOrigin/uBlock-issues#2289] [Q&A] [answered]
today Zemana says this path inside UBO is a trojan..Trojan:Win32/MalDropper.OnClickAds  ...how come?
Path= c:\users\debi\appdata\local\google\chrome\user 
data\default\extensions\cjpalhdlnbpafiamejdnhcphjbkeiagm\1.44.4_1\assets\thirdparties\pgl.yoyo.org\as\serverlist

**Comment by gorhill** [comment, marked as answer]
It's a false positive, it's just a filter list ([Peter Lowe's ads and trackers](https://pgl.yoyo.org/adservers/)) containing names of remote server to be blocked. uBO contains the default set of filter lists in its package, this can cause overzealous anti-virus to be triggered.

**Reply by debifox** [reply]
ok, then I can remove it from quarantine,thx
